lib/usd/utils/CMakeLists.txt

```diff
@@ -84,3 +90,87 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
             DESTINATION ${CMAKE_INSTALL_PREFIX}/lib OPTIONAL
     )
 endif()
+
+# -----------------------------------------------------------------------------
+# sources
+# -----------------------------------------------------------------------------
+
+add_library(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME} SHARED)
+
+target_sources(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+PRIVATE
+  module.cpp
+  wrapMayaTransformAPI.cpp
+)
+
+# -----------------------------------------------------------------------------
+# python init
+# -----------------------------------------------------------------------------
+
+set(PY_INIT_FILES
+  __init__.py
+)
+
+mayaUsd_copyFiles(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+  DESTINATION 
+    ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME}
+  FILES
+    ${PY_INIT_FILES}
+)
+
+# -----------------------------------------------------------------------------
+# compiler configuration
+# -----------------------------------------------------------------------------
+target_compile_definitions(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+  PRIVATE
+    MFB_PACKAGE_NAME=${TARGET_NAME}
+    MFB_ALT_PACKAGE_NAME=${TARGET_NAME}
+    MFB_PACKAGE_MODULE=${TARGET_NAME}
+)
+
```

Comment:
@robthebloke  does this branch even compiles anywhere?

You need to add this after :  `mayaUsd_compile_config(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME})`